tdrs-frontend/package.json

```diff
@@ -6,20 +6,20 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.36",
     "@fortawesome/free-solid-svg-icons": "^5.15.4",
     "@fortawesome/react-fontawesome": "^0.1.17",
+    "@lagunovsky/redux-react-router": "^2.2.0",
     "axios": "^0.25.0",
     "classnames": "^2.3.0",
-    "connected-react-router": "^6.8.0",
     "file-type": "^16.5.3",
-    "history": "^4.7.2",
+    "history": "^5.2.0",
```

Comment:
This was updated as the older version caused a conflict with the new one needed in react-router-dom